tests/test_integration.py

```diff
@@ -0,0 +1,157 @@
+import dash
+from dash import html, Input, Output, State
+from dash_player import DashPlayer
+
+
+def test_001_child_with_0(dash_duo):
```

Comment:
is this test an artifact of figuring out how to write tests?

Reply:
Yes, I'll remove that one